### PR TITLE
Deezer and search command

### LINKED ISSUE #667: Subcommands load errors
**Bug description**
When i try to start the bot, it gives a load error when loading a subcommand that the parent command cant load(because of a requeriment of an api for example)

**How to reproduce**
1. Create a command that cant load, an api requeriment without the api on env vars for example
2. Create a subcommand with the parent command as the last step
3. Start the bot

**Expected behavior**
No errors

**Screenshots**
image

### PR DESCRIPTION
**Pull request by metehus**
### Deezer command
- [x] Searches for a track
- [x] Searches for an album
- [x] Searches for an artist
- [x] Searches for an user
- [x] Searches for a playlist
- [x] Searches for a podcast
- [x] Add `--tracks`, for a playlist and album
- [x] Add `--albums` and `--related` for an artist
- [x] Add `--episodes` for a podcast
#### Search command
Search handler to easily make search commands
- [x] Update spotify commands to the new Search command structure
- [x] Update last.fm commands to the new Search command structure 
- [x] Update github command to the new search command structure
~~Update Discord bot list command to the new search command structure~~ fuck pill
#### Fixes
- [x] Fix #667 

### [Screenshots](https://imgur.com/a/l9oyaqd)

### CODE REVIEW

**Comment by metehus** [comment]
GOOD LUCK

**Comment by metehus** [comment]
aooo cornada

**Comment by metehus** [comment]
FUCK WAIT

**Comment by metehus** [comment]
I DID A BIG SHIT FUCKKKK

**Comment by metehus** [comment]
FIXED THANKS @davipatury LORD

File: src/utils/MiscUtils.js [hunk 1]
```diff
@@ -12,4 +13,8 @@ module.exports = class MiscUtils {
     const formatter = new Intl.NumberFormat(language)
     return formatter.format(value)
   }
+
+  static formatDuration (duration) {
+    return moment.duration(duration).format('mm:ss')
```
Comment (davipatury) [inline src/utils/MiscUtils.js]: What about durations longer than 59 minutes?

File: src/commands/misc/lastfm/user.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    const embed = new SwitchbladeEmbed(author)
+
+    try {
+      const { user } = await this.client.apis.lastfm.getUserInfo(param)
+      const time = moment(user.registered.unixtime * 1000).format('LLL')
+
+      let { topartists } = await this.client.apis.lastfm.getUserTop(param, 'artists', '1month', 5)
+      topartists = topartists.artist
+
+      embed.setAuthor(user.realname ? user.realname : user.name, 'https://i.imgur.com/TppYCun.png', user.url)
```
Comment (davipatury) [inline src/commands/misc/lastfm/user.js]:
```suggestion
      embed.setAuthor(user.realname || user.name, 'https://i.imgur.com/TppYCun.png', user.url)
```

**Comment by Doges** [comment]
i m just mining

**Comment by metehus** [comment]
@Doges aceita ae irmao

**Pull request merged**